app/models/sail/setting.rb

```diff
@@ -79,6 +81,12 @@ def value_is_within_range
       end
     end
 
+    def date_is_valid
+      DateTime.strptime(value, '%Y-%m-%d')
```

Comment:
DateTime.parse can infer the date's format from the input. Any reason on why we would limit it to year-month-day?

Reply:
@vinistock  Yeah agree. Somehow I overlooked this thinking that we don't need time part of the DateTime but it does not matter. I have fixed it and pushed a commit now.